docs/features.rst

```diff
@@ -25,11 +25,9 @@ compliant way, by just running::
     tox -e build
 
 Alternatively, if you are not a huge fan of isolated builds, or prefer running
-the commands yourself, you can execute ``python setup.py bdist_wheel``. Source
-distributions, i.e. ``sdist``, are obsolete and no longer recommended as they
-ignore several options in ``setup.cfg``. If ``universal=1`` in the ``[bdist_wheel]``
-section of ``setup.cfg``, a generated wheel distribution will be as flexible as
-a source distribution and can be installed on every architecture.
+the commands yourself, you can execute ``python -m build --no-isolation .``.
+Source distributions, i.e. ``sdist``, are only recommended if you absolutely need them.
+They are tricky to configure and may ignore several options in ``setup.cfg``.
```

Comment:
Should we keep saying `sdist` are deprecated? `bdist` definetely are, but the `build` package and PEP517 seem to be fine with `sdist` (or at least do not say officially they are deprecated).

Regardless, since `sdist`s are at least tricky, should we change the `tox.ini` template to not build them by default?

Reply:
Great suggestion. I think the default should be wheel-only. I see no reason why one needs source dists and it's good that we tell at that point that they are tricky and ignore options.

Reply:
Should we also only release wheels for PyScaffold itself in the future? (So I can apply the same changes for `tox.ini` and `tox_ini.template`? 😝

Reply:
Yes, that makes sense! We needed this only until 3.x since setuptools was a development requirement until then, i.e. ``setup_requires``. As this is no longer needed we should drop it and also remove all the notes about ``python setup.py egg_info --egg-base .`` as you mentioned above.